bot.js

```diff
@@ -152,6 +149,26 @@ client.on('message', message => {
 		});
 	}
 
+	if(message.content.substring(0,6) === `${prefix}class`){
+		const args = message.content
+		const req = args.split(" ").length === 3 ? `${args.split(" ")[1]}${args.split(" ")[2]}`:args.split(" ")[1]
+		const roster = getRoster()
+		const member = roster.find(member => member.name.toLowerCase() === req.toLowerCase())
+
+		if(member && member.name.toLowerCase().includes(req.toLowerCase())){
+			message.channel.send(`${member.name} plays ${member.role.substring(0,1).toUpperCase() + member.role.substring(1)}. That's a really cool class!`)
+			return;
+		}
+
+		if(!MAPLE_STORY_CLASSES.includes(req.toLowerCase())){
+			message.channel.send(`Zakum tried his best but does not think that ${req} exists in MapleStory M.`)
+			return;
+		}
+
+		const classList = roster.filter(member => member.role.toLowerCase() === req.toLowerCase()).map(member => member.name)
+		classList && classList.length ? message.channel.send(`${classList.toString()} (${classList.length})`) : message.channel.send(`Zakum could not locate any guild members that play ${req}.`)
```

Comment:
classList.toString() => classList.join(', ') //c'mon man, you're better than this. lol

Reply:
why you gotta roast me like that :(

Reply:
sorry sorry